### ISSUE TITLE
VRAM Requirements to Fine-tune SVD

### ISSUE THREAD

**Issue by mlfarinha** [alibaba/animate-anything#21]
Hi,
Found your repo very useful.

I was wondering whether you know how much VRAM is necessary to train/fine-tune [SVD](https://huggingface.co/stabilityai/stable-video-diffusion-img2vid-xt) in the normal setting (without mask)?

Thank you,
Miguel

**Comment by daizuozhuo** [comment]
Finetuning the whole model requires 80G VRAM. If you just finetune the temporal layers, and reduce the resolution, 32G is also enough.
